src/content/finite-state-machine.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: post
+title: "Finite State Machine trong web development"
+author: Dr.Yami
+tags: ["Programming"]
```

Comment:
@suppayami Please find a better tags for this post :D We will have a brazillion of `Programming` post later